workflows/promote-nightly.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Promotes a nightly release.
+ */
+
+import {createVersionsUpdatePullRequest, runPromoteJob} from './promote-job';
+
+const jobName = 'promote-nightly.js';
+const {AMP_VERSION} = process.env;
+
+void runPromoteJob(jobName, async () => {
```

Comment:
TIL the void operator. We don't want to return undefined right? It would be better to return the promise

Reply:
Oh yeah I should've mentioned that. Without the `void` operator I get an eslint error:
```
Promises must be awaited, end with a call to .catch, end with a call to .then
with a rejection handler or be explicitly marked as ignored with the `void`
operator. eslint@typescript-eslint/no-floating-promises
```

We can't await on the top level without jumping through a lot of hoops, but this is okay because Node will wait until all Promises are resolved/rejected before exiting, so there's no need to await this explicitly (though I prefer if we could do that...)

Reply:
Would this work?
```
function runPromoteJob() {

  ...
  
  return createVersionsUpdatePullRequest()
}

runPromoteJob();
```

Reply:
No, because that creates a function that returns a Promise, and then we just have the same problem :)

Reply:
Ok then! void it is